DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: inline
-Version: 0.3.15
-Date: 2018-05-18
+Version: 0.3.16
+Date: 2020-07-11
```

Comment:
Please revert that to (at the most) 0.3.15.1 and current date.

Setting releases is what Maintainer: does.

Reply:
Also rebasing/squashing may be a good idea though I can do that too when I merge.